dashboard/cron.py

```diff
@@ -387,6 +387,69 @@ def weight_consideration(self):
 
         return status
 
+    def update_group_membership(self):
+        #load group membership
+        status =""
+
+        # delete all records in group_membership table
+        status += deleteAllRecordInTable("group_membership")
+
+        # loop through multiple course ids
+        for data_warehouse_course_id in Course.objects.get_supported_courses():
+            group_membership_url = f"""
+                with grp_info as (select gf.group_id
+                                  from group_fact gf join group_dim gd on gf.group_id = gd.id
+                                  where gf.parent_course_id = '{data_warehouse_course_id}' and gd.workflow_state = 'available'),
+                grp_member_info as (select gmf.group_id, gmf.user_id, gmf.group_membership_id
+                                    from group_membership_fact gmf join grp_info gi on gmf.group_id = gi.group_id)
+                select gmi.group_id, gmi.user_id
+                from group_membership_dim gmd join grp_member_info gmi on gmd.id = gmi.group_membership_id
+                where workflow_state = 'accepted'
+            """
+            status += util_function(data_warehouse_course_id, group_membership_url, 'group_membership')
+            logger.debug(status+"\n\n")
+
+        return status
+
+    def update_discussion(self):
+        #load the discussion topic and entry information within a course into a flat table.
+        status =""
+
+        # delete all records in discussion_flattened table
+        status += deleteAllRecordInTable("discussion_flattened")
+
+        # loop through multiple course ids
+        for data_warehouse_course_id in Course.objects.get_supported_courses():
+            # NOTE: Pandas has trouble with int columns with null values mixed in (converts to float causing perscision issues)
+            # as a way around this, some fields (entry_id, assignment_id, group_id, and user_id) are cast as varchar in the query
```

Comment:
Ya pandas caused the same issue with a specific case of assignment view [here](https://github.com/tl-its-umich-edu/my-learning-analytics/commit/e062ab7e9bd38a772d375adfd8f2d2faf3b58eaa#diff-474394b3595bf5cedbf0bf5ee1080b66R483) worked around was ifnull(assignment_id) then add a zero